DOCUMENTATION.md

```diff
@@ -107,19 +122,162 @@ In case it's a **dictionary** with fields, please make sure the **encoder** func
+```python
+client.register_device(dev_id, device)
+```
+- `dev_id`: **string**  the id of the device to be registered.
+- `device`: **dictionary**  the dictionary with fields to be set as a new device of the application. See the [Device](#device) section to know the structure of the dictionary that should be passed and the name of the fields.
+
+### device
+Gives back the [**Device**](#device) object of the given id.
+```python
+client.device(dev_id)
+```
+- `dev_id`: **string**  the id of the device which is given back by the method.
+
+### devices
+Gives back the list of all the devices registered to the application.
+```python
+client.devices()
+```
+
+### update_device
+Updates an already existing device of the application.
+```python
+client.update_device(dev_id, updates)
+```
+- `dev_id`: **string**  the id of the device to be updated.
+- `updates`: **dictionary**  a dictionary with the fields to be updated in the device.
+
+### delete_device
+Deletes the device with the given id.
+```python
+client.delete_device(dev_id)
+```
+- `dev_id`: **string**  the id of the device to be deleted.
+
+### Device
+This objet is returned by the method `device()` of the ApplicationClient class. Here are its attributes:
+* `app_id`: **string**
+* `dev_id`: **string**
+* `latitude`: **float**
+* `longitude`: **float**
+* `altitude`: **float**
+* `description`: **string**
+* `attributes`: **dictionary**
+* `lorawan_device`: **dictionary**
+    * `app_eui`: **string**  8 bytes in hexadecimal
+    * `dev_eui`: **string**  8 bytes in hexadecimal
+    * `dev_addr`: **string**  4 bytes in hexadecimal
+    * `nwk_s_key`: **string**  16 bytes in hexadecimal
+    * `app_s_key`: **string**  16 bytes in hexadecimal
+    * `app_key`: **string**  16 bytes in hexadecimal
+    * `f_cnt_up`: **int**
+    * `f_cnt_down`: **int**
+    * `disable_f_cnt_check`: **boolean**
+    * `uses32_bit_f_cnt`: **boolean**
+
+### Application
+This object is returned by the method `get()` of the ApplicationClient class. Here are its attributes:
+* `app_id`: **string**
+* `payload_format`: **string**
+* `decoder`: **string**
+* `encoder`: **string**
+* `converter`: **string**
+* `validator`: **string**
+* `register_on_join_access_key`: **string**
+
+## HandlerClient
+The class constructor can be called following this scheme:
+```python
+HandlerClient(app_id, access_key_or_token, [discovery_address], [certificate])
+```
+- `app_id`: **string**  this the name given to the application when it was created.
+![Screenshot of the console with app section](./images/app-console.png?raw=true)
+- `app_access_key`: **string**  this can be found at the bottom of the application page under **ACCESS KEYS**. You will need a key allowing you to change the settings if you wish to update your application.
+- `discovery_address`: **string**  this is the address of the discovery server to use in order to find back the address of the handler to which the application in registered. Default to `None`.
```

Comment:
`None` is not a string. What happens if `discovery_address` is not set? Please document it.